app/services/symphony_client.rb

```diff
@@ -7,6 +7,11 @@ class SymphonyClient
     content_type: 'application/json'
   }.freeze
 
+  ITEM_RESOURCES = [
+    'bib{title,author,callList{*}}',
+    'item{*,bib{shadowed,title,author},call{sortCallNumber,dispCallNumber}}'
+  ].join(',')
+
```

Comment:
Moving this up with the rest of the constants makes sense I think. Why'd you turn it into an array with two values (bib and item,call)? I think either it should stay a simple string or maybe break it into bib, item and call (3 values of the array) to improve readability. Assuming this change had to be made due to line length?

Reply:
The only other thing I am struggling to wrap my head around with all of this now is why we are calling for title and author from `bib` and `item>bib`  and then (relatedly) pulling `shadowed` from `item>bib` (I know below for hold lookup and item lookup both are pulling `shadowed` from just the bib level).

Had to look again at checkouts and holds.

For checkouts, the `circRecordList`'s `circRecord`s do not contain `bib` (even though we ask for it as part of the web service request). It'll pass bib info along because we specified it in the call (i.e., `item{*,bib{shadowed,title,author}`).

For holds, the `holdRecordList`'s `holdRecord` _does_ contain `bib`. (aside, we could drop the ask for title and author from `bib` or `item>bib` on the holds page - right now I guess it's a bit redundant, but probably not a problem in terms of speed).

I didn't look at fines because the test account I had didn't have any. But it doesn't matter really because I see why we can't just always use "top-level" `bib`. Sometimes `bib` is available at the "top-level" and sometimes it isn't (like for `circRecord`) - *but* `bib` is always available at the `item > bib` level.

Hope this rambling made some sense, just had to walk back through the symphony request logic. No change asked for, to be clear.

Reply:
> Moving this up with the rest of the constants makes sense I think. Assuming this change had to be made due to line length?

yes, because of the line length rubocop was getting annoyed

Reply:
> 
> I didn't look at fines because the test account I had didn't have any. But it doesn't matter really because I see why we can't just always use "top-level" `bib`. Sometimes `bib` is available at the "top-level" and sometimes it isn't (like for `circRecord`) - _but_ `bib` is always available at the `item > bib` level.
> 
> Hope this rambling made some sense, just had to walk back through the symphony request logic. No change asked for, to be clear.

yes, sometimes it is not available at the top-level and that may be redundant really, let's  chat about this to see if we can remove that part.

Reply:
Does sorta make me wonder if the dynamic loading of arguments to the web service is a little _too_ fancy and complicated. But, again, no change needed.